README.md

```diff
@@ -5,6 +5,17 @@
 <!-- Remove this if you don't use github actions -->
 ![Build and populate cache](https://github.com/dagger/nix/workflows/Build%20and%20populate%20cache/badge.svg)
 
+This is updated by [GoReleaser
+Nixpkgs](https://goreleaser.com/customization/nix/) from within the
+[dagger/dagger](https://github.com/dagger/dagger/blob/v0.11.7/.goreleaser.yml#L31-L48)
```

Comment:
Is the tag pin on purpose?

Reply:
I tend to do that so that links don't get broken when files move around. Would you prefer to stick with `main` ? I'm easy.

Reply:
I just wasn't sure if it was on purpose or not. I don't really mind either way.